### PR TITLE
Reduce size of logs of nakadi

### PR DESCRIPTION
**Pull request by antban**
ARUHA-1828

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/filters/LoggingFilter.java [hunk 1]
```diff
@@ -55,7 +56,7 @@ protected void doFilterInternal(final HttpServletRequest request,
                     .orElse("-");
             final Long contentLength = request.getContentLengthLong() == -1 ? 0 : request.getContentLengthLong();
 
-            LOG.info("[ACCESS_LOG] {} \"{}{}\" \"{}\" \"{}\" statusCode: {} {} ms \"{}\" \"{}\" {} bytes",
+            ACCESS_LOGGER.info("[ACCESS_LOG] {} \"{}{}\" \"{}\" \"{}\" statusCode: {} {} ms \"{}\" \"{}\" {} bytes",
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/filters/LoggingFilter.java]: @antban 
1) I think another quick win will be to make `[ACCESS_LOG]` string shorter
2) and also we can remove "statusCode:" string as our parser extract the actual value anyway
3) we can also replace `bytes` with 'B'

**Comment by antban** [comment]
deploy validation please

**Comment by v-stepanov** [comment]
deploy validation please

**Comment by v-stepanov** [comment]
deploy validation please

**Pull request merged**